README.md

```diff
@@ -36,9 +36,9 @@ There are two UNIX domain sockets used by the node-driver-registrar:
 * CSI driver socket:
   * Used by kubelet to interact with the CSI driver.
   * Created by the CSI driver.
-  * Exposed on a Kubernetes node via hostpath somewhere other than the Kubelet.
-    plugin registry (typically `/var/lib/kubelet/plugins/<drivername.example.com>/csi.sock`).
+  * Exposed on a Kubernetes node via hostpath somewhere other than the Kubelet plugin registry. (typically `/var/lib/kubelet/csi-plugins/<drivername.example.com>/csi.sock`).
   * This is the socket referenced by the `--csi-address` and `--kubelet-registration-path` arguments.
+  * Note that if the csi socket is in the `/var/lib/kubelet/plugins/` path, kubelet may log a lot of harmful errors regarding grpc `GetInfo` call not implemented (see kubernetes/kubernetes#70485)
```

Comment:
This is resolved in 1.17.  We have removed CSI 0.3 support and therefore also removed watching "/var/lib/kubelet/plugins" for plugin registration sockets.  So from 1.17+, the "/var/lib/kubelet/plugins" directory won't cause log spam.

Also, I don't think "harmful" is right. It's not actually causing anything to error.

Reply:
See https://github.com/kubernetes/kubernetes/pull/84533

Reply:
Sorry I meant harmless :sweat_smile:

Reply:
Okay. I'll change the phrasing :)

Reply:
@msau42 better now?